src/amber/cli/templates/app/shard.yml.ecr

```diff
@@ -18,12 +18,12 @@ targets:
 dependencies:
   amber:
     github: amberframework/amber
-    branch: master
+    branch: dj-crystal-34-support
```

Comment:
changed this to use this branch so tests will pass.  we need to change this back to master.